src/effects/Distortion.h

```diff
@@ -239,20 +250,33 @@ class EffectDistortion final : public StatefulPerTrackEffect
    static const EnumValueSymbol kTableTypeStrings[nTableTypes];
 
 // (Note: 'Repeats' is the total number of times the effect is applied.)
-static constexpr EnumParameter TableTypeIndx{ &EffectDistortion::Params::mTableChoiceIndx,
-   L"Type",           0,       0,      nTableTypes-1,    1, kTableTypeStrings, nTableTypes    };
-static constexpr EffectParameter DCBlock{ &EffectDistortion::Params::mDCBlock,
-   L"DC Block",      false,   false,   true,                1    };
-static constexpr EffectParameter Threshold_dB{ &EffectDistortion::Params::mThreshold_dB,
-   L"Threshold dB",  -6.0,  -100.0,     0.0,             1000.0f };
-static constexpr EffectParameter NoiseFloor{ &EffectDistortion::Params::mNoiseFloor,
-   L"Noise Floor",   -70.0,  -80.0,   -20.0,                1    };
-static constexpr EffectParameter Param1{ &EffectDistortion::Params::mParam1,
-   L"Parameter 1",    50.0,    0.0,   100.0,                1    };
-static constexpr EffectParameter Param2{ &EffectDistortion::Params::mParam2,
-   L"Parameter 2",    50.0,    0.0,   100.0,                1    };
-static constexpr EffectParameter Repeats{ &EffectDistortion::Params::mRepeats,
-   L"Repeats",        1,       0,       5,                  1    };
+static constexpr EnumParameter TableTypeIndx
+{ &EffectDistortionSettings::mTableChoiceIndx, L"Type",
+   EffectDistortionSettings::mDefaultTableChoiceIndx,    0,      nTableTypes-1,    1, kTableTypeStrings, nTableTypes    };
+
+static constexpr EffectParameter DCBlock
+{ &EffectDistortionSettings::mDCBlock, L"DC Block",
+   EffectDistortionSettings::mDefaultDCBlock ,   false,   true,                1    };
+
+static constexpr EffectParameter Threshold_dB
+{ &EffectDistortionSettings::mThreshold_dB, L"Threshold dB",
+   EffectDistortionSettings::mDefaultThreshold_dB,  -100.0,     0.0,             1000.0f };
+
+static constexpr EffectParameter NoiseFloor
+{ &EffectDistortionSettings::mNoiseFloor, L"Noise Floor",
+   EffectDistortionSettings::mDefaultNoiseFloor ,  -80.0,   -20.0,                1    };
+
+static constexpr EffectParameter Param1
+{ &EffectDistortionSettings::mParam1, L"Parameter 1",
+   EffectDistortionSettings::mDefaultParam1,    0.0,   100.0,                1    };
+
+static constexpr EffectParameter Param2
+{ &EffectDistortionSettings::mParam2, L"Parameter 2",    50.0,    0.0,   100.0,                1    };
```

Comment:
Replace 50.0 with the named constant `mDefaultParam2` consistently with the other parameters

Reply:
good catch, done